crates/net/network/src/config.rs

```diff
@@ -69,6 +69,19 @@ pub struct NetworkConfig<C> {
     pub status: Status,
     /// Sets the hello message for the p2p handshake in RLPx
     pub hello_message: HelloMessage,
+    /// Whether to disable transaction gossip
+    pub tx_gossip_disabled: bool,
+    /// Optimism Network Config
+    #[cfg(feature = "optimism")]
+    pub optimism_network_config: OptimismNetworkConfig,
+}
+
+/// Optimmism Network Config
+#[cfg(feature = "optimism")]
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Default)]
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/149